app/views/rubygems/_aside.html.erb

```diff
@@ -51,6 +51,15 @@
     </i>
   </h2>
 
+  <% if @rubygem.mfa_required? %>
+    <h2 class="gem__ruby-version__heading t-list__heading">
+      <%= t('.requires_mfa') %>:
+      <span class="gem__ruby-version">
+        <%= t('.since', version: @rubygem.mfa_required_since_version) %>
```

Comment:
This is more informative 👍 

Could this still produce misleading info, say if MFA-required is turned on, then off, and back on again?

What would this show if the current version is 1.0.8 and the below is the MFA-required history?

- v1.0.0-1.0.2 MFA required
- v1.0.3-1.0.5 MFA not required
- v1.0.6-1.0.8 MFA required

Reply:
@eliotsykes I've fixed the implementation to cover the case you described - it will now show "since 1.0.6" which is the correct information in my view. I've also added this case to integration test.